hoomd/device.py

```diff
@@ -271,6 +272,29 @@ def is_available():
         """
         return _hoomd.isCUDAAvailable();
 
+    @contextlib.contextmanager
+    def enable_profiling(self):
+        """ Enable GPU profiling.
+
+        When using GPU profiling tools on HOOMD, select the option to disable
+        profiling on start. Initialize and run a simulation long enough that
```

Comment:
Select the option to disable profiling on start? Is there an option elsewhere that this should link to? I am not aware of other ways to enable profiling, so maybe I'm missing something.

Reply:
The option is tool specific. For example, there is a checkbox in the nvvp project setup, there is a command line option for nvprof. I don't recall where the option is in NSight systems and I've never used the AMD profiling tools myself.

Reply:
Nsight has the same option, and IIRC it can be changed mid-run unlike nvvp (where it has to be specified when you create the workspace).